stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/ChatErrorTest.kt

```diff
@@ -3,38 +3,33 @@ package io.getstream.chat.android.livedata
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.google.common.truth.Truth
 import io.getstream.chat.android.client.errors.ChatNetworkError
-import io.getstream.chat.android.client.models.Message
 import io.getstream.chat.android.livedata.extensions.isPermanent
 import org.junit.Test
 import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
```

Comment:
It's no longer an integration test. Why are we still running it with instrumented test runner? @leandroBorgesFerreira